src/services/refactors/moveToFile.ts

```diff
@@ -247,10 +247,10 @@ function getNewStatementsAndRemoveFromOldFile(
             moveStatementsToTargetFile(changes, program, body, targetFile, toMove);
         }
         else {
-            changes.insertNodesAtEndOfFile(targetFile, body, /*blankLineBetween*/ false);
+            changes.insertNodesAtEndOfFile(targetFile, body, /*blankLineSuffix*/ false, !imports || usage.oldImportsNeededByTargetFile.size === 0 ? true : false);

```

Comment:
Can `imports` have length 0?
Could you add a comment explaining when we want to add a blank line prefix?